lib/bibliothecary/parsers/maven.rb

```diff
@@ -109,6 +115,26 @@ def self.parse_gradle_resolved(file_contents)
         end.compact.uniq {|item| [item[:name], item[:requirement], item[:type]]}
       end
 
+      def self.parse_maven_resolved(file_contents)
+        file_contents
+          .gsub(/#{ANSI_MATCHER}/, '')
```

Comment:
Use `gsub` to remove ANSI characters from the file string.

Reply:
You can make the constant the regex itself, and not have to do this interpolation